test/flipper_app/authorize_test.rb

```diff
@@ -0,0 +1,75 @@
+require_relative '../test_helper'
+require 'mocha/setup'
+require 'flipper_app/authorize'
+
+class AssignmentsApiTest < Minitest::Test
+  def test_unauthenticated_users_rejected
+    User.stubs(:where).returns(authorized_users)
```

Comment:
It fix one problem, but there is also the case where if you chain this relation with another method (`where`, `order`, etc..) you will get a new relation that is not stubbed. It is usually better to stub the `API` that we own (`authorized_github_ids` in this case)  than stub an API of an object that we don't have any control over. :)